packages/ui/src/components/Table.svelte

```diff
@@ -106,7 +122,11 @@
                     <div class="menuBtn"><MoreV size={16} /></div>
                   </div>
                 {/if}
-                <svelte:component this={cell.component} {...cell.props} />
+                {#if typeof cell.component === 'string'}
+                  <Component is={asComponent(cell.component)} props={cell.props} />
```

Comment:
Let's check if this construction is used in any other place and if so, let's move this logic inside Component itself.